2021-11-22 01:25:26,756 INFO    MainThread:128129 [wandb_setup.py:_flush():71] setting env: {'project': '11_21train'}
2021-11-22 01:25:26,757 INFO    MainThread:128129 [wandb_setup.py:_flush():71] setting login settings: {}
2021-11-22 01:25:26,757 INFO    MainThread:128129 [wandb_init.py:_log_setup():371] Logging user logs to /home/yu-jw19/venom/project2/wandb/run-20211122_012526-3k5lj6ff/logs/debug.log
2021-11-22 01:25:26,757 INFO    MainThread:128129 [wandb_init.py:_log_setup():372] Logging internal logs to /home/yu-jw19/venom/project2/wandb/run-20211122_012526-3k5lj6ff/logs/debug-internal.log
2021-11-22 01:25:26,758 INFO    MainThread:128129 [wandb_init.py:init():404] calling init triggers
2021-11-22 01:25:26,758 INFO    MainThread:128129 [wandb_init.py:init():409] wandb.init called with sweep_config: {}
config: {}
2021-11-22 01:25:26,758 INFO    MainThread:128129 [wandb_init.py:init():449] starting backend
2021-11-22 01:25:26,758 INFO    MainThread:128129 [backend.py:_multiprocessing_setup():95] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-11-22 01:25:26,779 INFO    MainThread:128129 [backend.py:ensure_launched():199] starting backend process...
2021-11-22 01:25:26,790 INFO    MainThread:128129 [backend.py:ensure_launched():204] started backend process with pid: 128198
2021-11-22 01:25:26,792 INFO    MainThread:128129 [wandb_init.py:init():458] backend started and connected
2021-11-22 01:25:26,794 INFO    MainThread:128129 [wandb_init.py:init():519] updated telemetry
2021-11-22 01:25:26,811 INFO    MainThread:128129 [wandb_init.py:init():549] communicating current version
2021-11-22 01:25:27,967 INFO    MainThread:128129 [wandb_init.py:init():554] got version response 
2021-11-22 01:25:27,967 INFO    MainThread:128129 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2021-11-22 01:25:28,529 INFO    MainThread:128129 [wandb_init.py:init():592] starting run threads in backend
2021-11-22 01:25:29,344 INFO    MainThread:128129 [wandb_run.py:_console_start():1816] atexit reg
2021-11-22 01:25:29,345 INFO    MainThread:128129 [wandb_run.py:_redirect():1690] redirect: SettingsConsole.REDIRECT
2021-11-22 01:25:29,345 INFO    MainThread:128129 [wandb_run.py:_redirect():1695] Redirecting console.
2021-11-22 01:25:29,347 INFO    MainThread:128129 [wandb_run.py:_redirect():1751] Redirects installed.
2021-11-22 01:25:29,347 INFO    MainThread:128129 [wandb_init.py:init():619] run started, returning control to user process
2021-11-22 01:25:29,356 INFO    MainThread:128129 [wandb_run.py:_config_callback():962] config_cb None None {'dataset': 'emotiondataset', 'name': 'firsttest_network=resnet,layers=32', 'no': '0', 'model': 'resnet', 'print_freq': 10, 'layers': 32, 'droprate': 0.3, 'cardinality': 8, 'growth_rate': 12, 'compression_rate': 0.5, 'bn_size': 4, 'alpha': 200, 'N': 2, 'M': 10, 'augment': True, 'randaugment': True, 'checkpoint': 'checkpoint', 'resume': False, 'cos_lr': True, 'initial_learning_rate': 0.1, 'changing_lr': [80, 120], 'en_wandb': True, 'warm_up': False, 'exp_id': '00041_network=resnet,layers=32', 'commit': 'not_set'}
2021-11-22 01:25:57,206 INFO    MainThread:128129 [wandb_run.py:_atexit_cleanup():1786] got exitcode: 1
2021-11-22 01:25:57,206 INFO    MainThread:128129 [wandb_run.py:_restore():1758] restore
2021-11-22 01:25:59,690 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-22 01:25:59,796 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-22 01:26:00,313 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 970
}

2021-11-22 01:26:00,579 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9870
}

2021-11-22 01:26:00,682 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9870
}

2021-11-22 01:26:00,786 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9870
}

2021-11-22 01:26:00,889 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9870
}

2021-11-22 01:26:00,992 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9870
}

2021-11-22 01:26:01,096 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 970
  total_bytes: 9870
}

2021-11-22 01:26:01,199 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9843
  total_bytes: 9870
}

2021-11-22 01:26:01,302 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9870
  total_bytes: 9870
}

2021-11-22 01:26:01,405 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9870
  total_bytes: 9870
}

2021-11-22 01:26:01,509 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9870
  total_bytes: 9870
}

2021-11-22 01:26:01,611 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9870
  total_bytes: 9870
}

2021-11-22 01:26:01,715 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9870
  total_bytes: 9870
}

2021-11-22 01:26:01,818 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9870
  total_bytes: 9870
}

2021-11-22 01:26:02,127 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9870
  total_bytes: 9870
}

2021-11-22 01:26:03,197 INFO    MainThread:128129 [wandb_run.py:_wait_for_finish():1916] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 9870
  total_bytes: 9870
}
local_info {
}

2021-11-22 01:26:04,300 INFO    MainThread:128129 [wandb_run.py:_append_files():2184] logging synced files
